Examples/Examples/ContentView.swift

```diff
@@ -103,3 +124,29 @@ private extension WebViewProxy {
 #Preview {
     ContentView()
 }
+
+extension PDFDocument: @retroactive Transferable {
+    public static var transferRepresentation: some TransferRepresentation {
+        DataRepresentation(contentType: .pdf) { pdf in
+            if let data = pdf.dataRepresentation() {
+                return data
+            } else {
+                return Data()
+            }
+        } importing: { data in
+            if let pdf = PDFDocument(data: data) {
+                return pdf
+            } else {
+                return PDFDocument()
+            }
+        }
+
+        DataRepresentation(exportedContentType: .pdf) { pdf in
+            if let data = pdf.dataRepresentation() {
+                return data
+            } else {
+                return Data()
+            }
+        }
```

Comment:
Since this is just a sample code, the Nil-Coalescing Operator might be fine.

```suggestion
        DataRepresentation(contentType: .pdf) { pdf in
            pdf.dataRepresentation() ?? .init()
        } importing: { data in
            PDFDocument(data: data) ?? .init()
        }

        DataRepresentation(exportedContentType: .pdf) { pdf in
            pdf.dataRepresentation() ?? .init()
        }
```